README.md

```diff
@@ -1,20 +1,20 @@
-# lnd - Lightning Network Daemon
+# Lightning Network Daemon
```

Comment:
`lnd` here should be kept, it's the name of the daemon. All other swaps below from `lnd` to `lightningd` should also be reverted.